models/ontology/taotest.rdf

```diff
@@ -107,4 +123,13 @@
   <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAO.rdf#TranslationProgress">
     <rdfs:domain rdf:resource="http://www.tao.lu/Ontologies/TAOTest.rdf#Test"/>
   </rdf:Description>
+  <rdf:Description rdf:about="http://www.tao.lu/Ontologies/TAO.rdf#TranslationCompletion">
```

Comment:
@shpran I think the same as here https://github.com/oat-sa/extension-tao-test/pull/476/files/556812e3dc2a3e479a6e837f9aff29e28846a30b..2c62e4d30448e64b2f8f702ca4d8a631870d8e4b#r1750050841